src/sentry/search/events/fields.py

```diff
@@ -2370,10 +2370,13 @@ def resolve_orderby(self, orderby: Optional[Union[List[str], str]]) -> List[Orde
                 ):
                     validated.append(OrderBy(selected_column, direction))
                     continue
-
-            # TODO: orderby aggregations
-
-            # TODO: orderby field aliases
+                elif (
+                    isinstance(selected_column, Function)
+                    and is_function(bare_orderby)
+                    and selected_column.alias == get_function_alias(bare_orderby)
```

Comment:
If I'm not wrong, the endpoint expects that the sort parameter is the alias of the column to sort by, so this is not covering that case.